src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,24 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
-);
+export const CommentInfo = ({ comment }) => {
+  const { name, email, body } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <span className="CommentInfo__name">{name}</span>
+
+        {' by '}
```

Comment:
```suggestion
        <span>by</span>
```
comment meant wrap this text in span